### ISSUE TITLE
.

### ISSUE THREAD

**Issue by ksh97** [GriefPrevention/GriefPrevention#67]
.

**Comment by NoblessHamster** [comment]
When I install griefprevention 16.4 and restart the Server, It works smoothly fine.

**Comment by NoblessHamster** [comment]
I think this crash is happen only 16.5

**Comment by Vestle** [comment]
correct I get this in latest 16.5
: The server has stopped responding!
[18:36:48] [Spigot Watchdog Thread/ERROR]: Please report this to http://www.spigotmc.org/
[18:36:48] [Spigot Watchdog Thread/ERROR]: Be sure to include ALL relevant console errors and Minecraft crash reports
[18:36:48] [Spigot Watchdog Thread/ERROR]: Spigot version: git-Spigot-7d78b81-775f1e1 (MC: 1.11.2)
[18:36:48] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:48] [Spigot Watchdog Thread/ERROR]: Server thread dump (Look for plugins here before reporting to Spigot!):
[18:36:48] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:48] [Spigot Watchdog Thread/ERROR]: Current Thread: Server thread
[18:36:48] [Spigot Watchdog Thread/ERROR]: 	PID: 17 | Suspended: false | Native: false | State: BLOCKED
[18:36:48] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.DataStore.getPlayerData(DataStore.java:529)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerInteractEntity(PlayerEventHandler.java:1114)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.GeneratedMethodAccessor443.invoke(Unknown Source)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		java.lang.reflect.Method.invoke(Method.java:498)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		com.massivecraft.massivelag.inject.MassiveLagEventExecutor.execute(MassiveLagEventExecutor.java:67)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1527)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PacketPlayInUseEntity.a(SourceFile:69)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PacketPlayInUseEntity.a(SourceFile:13)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.FutureTask.run(FutureTask.java:266)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:739)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:675)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:574)
[18:36:48] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:48] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:48] [Spigot Watchdog Thread/ERROR]: Entire Thread Dump:
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Async Chat Thread - #16
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 749 | Suspended: false | Native: false | State: BLOCKED
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerChat(PlayerEventHandler.java:115)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.GeneratedMethodAccessor729.invoke(Unknown Source)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.reflect.Method.invoke(Method.java:498)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		com.massivecraft.massivelag.inject.MassiveLagEventExecutor.execute(MassiveLagEventExecutor.java:60)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:484)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.chat(PlayerConnection.java:1266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1204)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:39)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.FutureTask.run(FutureTask.java:266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Async Chat Thread - #15
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 748 | Suspended: false | Native: false | State: BLOCKED
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerChat(PlayerEventHandler.java:115)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.GeneratedMethodAccessor729.invoke(Unknown Source)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.reflect.Method.invoke(Method.java:498)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		com.massivecraft.massivelag.inject.MassiveLagEventExecutor.execute(MassiveLagEventExecutor.java:60)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:484)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.chat(PlayerConnection.java:1266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1204)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:39)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.FutureTask.run(FutureTask.java:266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Async Chat Thread - #14
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 697 | Suspended: false | Native: false | State: BLOCKED
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Thread is waiting on monitor(s):
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		Locked on:me.ryanhamshire.GriefPrevention.DataStore.getMessage(DataStore.java:1690)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		Locked on:me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerChat(PlayerEventHandler.java:124)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:486)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.DataStore.getMessage(DataStore.java:1690)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.PlayerEventHandler.handlePlayerChat(PlayerEventHandler.java:254)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerChat(PlayerEventHandler.java:124)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.GeneratedMethodAccessor729.invoke(Unknown Source)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.reflect.Method.invoke(Method.java:498)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		com.massivecraft.massivelag.inject.MassiveLagEventExecutor.execute(MassiveLagEventExecutor.java:60)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:484)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.chat(PlayerConnection.java:1266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1204)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:39)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.FutureTask.run(FutureTask.java:266)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: File IO Thread
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 129 | Suspended: false | Native: false | State: TIMED_WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.sleep(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.FileIOThread.c(SourceFile:51)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.FileIOThread.run(SourceFile:30)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: pool-13-thread-1
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 127 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Chunk I/O Executor Thread-1
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 123 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Thread-39
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 112 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Object.wait(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.join(Thread.java:1245)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.join(Thread.java:1319)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.mrCookieSlime.CSCoreLibPlugin.updater.Updater$UpdaterTask.check(Updater.java:122)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		me.mrCookieSlime.CSCoreLibPlugin.updater.Updater$UpdaterTask.run(Updater.java:59)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Head Conversion Thread - 2
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 110 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Head Conversion Thread - 1
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 109 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Head Conversion Thread - 0
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 108 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Craft Scheduler Thread - 3
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 85 | Suspended: false | Native: false | State: WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Object.wait(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Object.wait(Object.java:502)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.primesoft.asyncworldedit.directChunk.relighter.BlockReligher.relightLoop(BlockReligher.java:312)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.primesoft.asyncworldedit.directChunk.relighter.BlockReligher.access$100(BlockReligher.java:63)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.primesoft.asyncworldedit.directChunk.relighter.BlockReligher$3.run(BlockReligher.java:235)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftTask.run(CraftTask.java:71)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Query Listener #1
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 81 | Suspended: false | Native: false | State: BLOCKED
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		org.bukkit.plugin.SimplePluginManager.getPlugins(SimplePluginManager.java:365)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.DedicatedServer.getPlugins(DedicatedServer.java:649)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.RemoteStatusListener.b(SourceFile:183)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.RemoteStatusListener.a(SourceFile:127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		net.minecraft.server.v1_11_R1.RemoteStatusListener.run(SourceFile:274)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Timer-3
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 78 | Suspended: false | Native: false | State: TIMED_WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Object.wait(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.TimerThread.mainLoop(Timer.java:552)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.TimerThread.run(Timer.java:505)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: Timer-2
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 73 | Suspended: false | Native: false | State: TIMED_WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Object.wait(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.TimerThread.mainLoop(Timer.java:552)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.TimerThread.run(Timer.java:505)
[18:36:49] [Spigot Watchdog Thread/ERROR]: ------------------------------
[18:36:49] [Spigot Watchdog Thread/ERROR]: Current Thread: MassiveLag StackTracer
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	PID: 68 | Suspended: false | Native: false | State: TIMED_WAITING
[18:36:49] [Spigot Watchdog Thread/ERROR]: 	Stack:
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		sun.misc.Unsafe.park(Native Method)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1093)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:809)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[18:36:49] [Spigot Watchdog Thread/ERROR]: 		java.lang.Thread.run(Thread.java:745)
[18:36:49] [Spigot Watchdog Thread/ERROR]: -------------------------

**Comment by RoboMWM** [comment]
Duplicate of #64 
Please include more of the log (lines before the watchdog thread dump) and add it to that issue.

**Comment by RoboMWM** [comment]
Duplicate of #64, please add your additional info to that ticket.
